modules/installation-gcp-permissions.adoc

```diff
@@ -23,32 +24,79 @@ install {product-title}. To deploy an {product-title} cluster, the service
 account requires the following permissions. If you deploy your cluster into an existing VPC, the service account does not require certain networking permissions, which are noted in the following lists:
 
 .Required roles for the installation program
-* Compute Admin
-* Security Admin
-* Service Account Admin
-* Service Account User
-* Storage Admin
+[cols="3,7"options="header"]
+|===
+|Role|Description
+
+|Compute Admin
+(roles/compute.admin)
+|link:https://cloud.google.com/iam/docs/understanding-roles#compute-engine-roles[Compute Engine roles]
+
+|Security Admin
+(roles/iam.securityAdmin)
+|Allows the service account to create, modify, and delete firewall rules and SSL certificates within the GCP project.
+
+|Service Account Admin
+(roles/iam.serviceAccountAdmin)
+|Creates and manages service accounts. This permission is required when managing your GCP credentials using mint mode.
+
+|Service Account User
+(roles/iam.serviceAccountUser)
+|Runs operations as the service account.
+
+|Storage Admin
+(roles/storage.admin)
+|Grants full control of storage objects and buckets, and is needed to help provide persistent storage to the cluster.
+|===
+
+For more information, see link:https://cloud.google.com/iam/docs/understanding-roles[Compute Engine roles].
 
 .Required roles for creating network resources during installation
-* DNS Administrator
+[cols="3,7"options="header"]
+|===
+|Role|Description
+
+|DNS Administrator
+(roles/dns.admin)
+|Provides the service account read-write access to all Cloud DNS resources. To install {product-title}, your GCP account must have a dedicated public hosted zone in the same project that you host the cluster. This zone must be authoritative for the domain. The DNS service provides cluster DNS resolution and name lookup for external connections to the cluster. A cluster also needs this permission to add new DNS entries for the Ingress Operator through the Kubernetes cloud integration.
```

Comment:
"A cluster also needs this permission..."? We are talking about a "role", rather than a "permission", right?